USING-CONFLUENCE.md

```diff
@@ -0,0 +1,69 @@
+# Using Confluence
+
+The confluence command uses the [Web API Confluence Dashboard](http://web-confluence.appspot.com/#!/) to generate new or improved browser compatibility data ready for submission in pull requests. The Web API Confluence Dashboard provides metrics and support data about web platform features across multiple browsers. The MDN Browser Compatibility Database provides a command for using Confluence to generate new and updated JSON files. This page explains how to do this.
+
+## Prerequisites
+
+Before using the confluence command make sure you have done all of the items below.
+
+* On GitHub fork this repo.
+* In this repo, add your fork as a remote repo: `git remote add fork-name path-to-fork`
+* In the root of your repo clone run `npm install`.
+* Configure a git [difftool](https://git-scm.com/docs/git-difftool).
+
+## Basic Procedure
```

Comment:
Other headings in this repo use sentence case, so I figure we may as well do the same

```suggestion
## Basic procedure
```